docs/sections/components/pipelines/index.md

```diff
@@ -216,6 +216,22 @@ sample_two_steps = sample_n_steps(2)
 
 ## Running the pipeline
 
+### Pipeline.dry_run
```

Comment:
Hi here @plaguss you may need to check your system settings, since every time you add the hash `#` the formatting gets messed up since a weird character is included afterwards

```suggestion
### Pipeline.dry_run
```

Reply:
Also note that otherwise I couldn't have created a suggestion on top of that, the issue cannot be noticed via the GitHub diff, but see that the space changed because from your system a weird symbol is included after `#` when you press space (I've seen this more times)